latest/index.bs

```diff
@@ -148,39 +133,47 @@ A well group SHOULD NOT be present if there are no images in the well.
 
 
 <pre>
-.                             # Root folder, potentially in S3,
+.
 │
-└── 5966.zarr                 # One plate (id=5966) converted to Zarr
-    ├── .zgroup
-    ├── .zattrs               # Implements "plate" specification
+└── 5966.zarr                 # One OME-Zarr plate (id=5966)
+    ├── zarr.json             # Implements "plate" specification
     ├── A                     # First row of the plate
-    │   ├── .zgroup
+    │   ├── zarr.json
     │   │
     │   ├── 1                 # First column of row A
-    │   │   ├── .zgroup
-    │   │   ├── .zattrs       # Implements "well" specification
+    │   │   ├── zarr.json     # Implements "well" specification
     │   │   │
     │   │   ├── 0             # First field of view of well A1
     │   │   │   │
-    │   │   │   ├── .zgroup
-    │   │   │   ├── .zattrs   # Implements "multiscales", "omero"
-    │   │   │   ├── 0
-    │   │   │   │   ...       # Resolution levels
-    │   │   │   ├── n
+    │   │   │   ├── zarr.json # Implements "multiscales", "omero"
+    │   │   │   ├── 0         # Resolution levels          
+    │   │   │   ├── ...
     │   │   │   └── labels    # Labels (optional)
-    │   │   ├── ...           # Fields of view
-    │   │   └── m
-    │   ├── ...               # Columns
-    │   └── 12
-    ├── ...                   # Rows
-    └── H
+    │   │   └── ...           # Other fields of view
+    │   └── ...               # Other columns
+    └── ...                   # Other rows
 </pre>
 
-Metadata {#metadata}
-====================
+OME-Zarr Metadata {#metadata}
+=============================
+
+The "OME-Zarr Metadata" contains metadata keys as specified below 
+for discovering certain types of data, especially images.
 
-The various `.zattrs` files throughout the above array hierarchy may contain metadata
-keys as specified below for discovering certain types of data, especially images.
+The OME-Zarr Metadata is stored in the various `zarr.json` files throughout the above array
+hierarchy. In this file, the metadata is stored under the namespaced key 
+`https://ngff.openmicroscopy.org/0.5` in `attributes`.
```

Comment:
Do we want to say anything here about whether we SHOULD or MUST use the same version within a hierarchy?
E.g. If Plate is v0.5, then can a reader assume that images within it are the same version?
I'm assuming we wouldn't want to support a Plate where some images are v0.4 (Zarr v2) and some are v0.5 (Zarr v3). This has probably always been an assumption by readers & writers but I'm not sure that the spec forbids it?